packages/next/src/auth/refresh.ts

```diff
@@ -17,6 +17,11 @@ export async function refresh({ config }: { config: any }) {
     throw new Error('Cannot refresh token: user not authenticated')
   }
 
+  const existingCookie = await getExistingAuthToken(payload.config.cookiePrefix)
+  if (!existingCookie) {
```

Comment:
I don't think this should be added - because the refresh operation is used by plugins that override all cookie / token-based behavior - so if you wanted to use the refresh op, but you weren't leveraging Payload's JWT, this would break that behavior

Reply:
interesting, maybe different thing that should be looked at separately